_sass/blocks/revenues/_revenues_subpage_steps.scss

```diff
@@ -129,11 +131,17 @@ $onshore-renewables-secondary-color: $blue;
     }
   }
 
-.revenues_subpage-steps_heading {
+.revenues_subpage-steps_heading,
```

Comment:
Selector `revenues_subpage-steps_heading` should be written in lowercase with hyphens<br>Selector `revenues_subpage-steps_heading_more` should be written in lowercase with hyphens